bundle/edu.gemini.pit/src/main/scala/edu/gemini/pit/ui/robot/ProblemRobot.scala

```diff
@@ -155,8 +155,8 @@ class ProblemRobot(s: ShellAdvisor) extends Robot {
       (t, Some(f)) <- s.catalogHandler.state
       (sev, msg) = f match {
         case Offline      => (Severity.Error, s"Catalog lookup failed for ${t.name} due to network connectivity problems.")
-        case NotFound(n)  => (Severity.Error, s"""Catalog lookup returned no results for target "$n" """)
-        case Error(e)     => (Severity.Error, s"Catalog lookup failed for ${t.name} due to an unexpected error: ")
+        case NotFound(n)  => (Severity.Error, s"""Catalog lookup returned no results for target "$n".""")
+        case Error(e)     => (Severity.Error, s"Catalog lookup failed for ${t.name} due to an unexpected error: ${e.getMessage}.")
```

Comment:
It seems, from what I can tell, that the ONLY time that this ever comes up is in `Catalog.empty`.
So you are probably right: we could eliminate the error and just put a period.